cmp/layout/Box.js

```diff
@@ -17,50 +16,33 @@ import {div} from './Tags';
  *
  * VBox and HBox variants support internal vertical (column) and horizontal (row) flex layouts.
  */
-@HoistComponent
-@LayoutSupport
-export class Box extends Component {
-    render() {
-        let {children, ...props} = this.getNonLayoutProps();
-        props = merge(
-            {style: {display: 'flex', overflow: 'hidden', position: 'relative'}},
-            {style: this.getLayoutProps()},
-            props
-        );
+export const [Box, box] = hoistComponent(function Box(props) {
```

Comment:
John:  named function here is for the benefit of how the component will appear in the React developer tools.  Otherwise our component will appear anonymous.  At Anselm's suggestion, I did this for a number of core low-level components.

Its actually annoying needing to specify the name of the component *three* times -- and spoiling the fun of removing the elemFactory() and its requirement to repeat the Component's naem.